daisy_integration_tests/create_random_data_disk.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+i# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -x
+
+BYTES_1GB=1073741824
+METADATA_URL="http://metadata/computeMetadata/v1/instance"
+DISK_DATA_NAME=$(curl -f -H Metadata-Flavor:Google ${METADATA_URL}/attributes/disk-data-name)
+SIZE=$(curl -f -H Metadata-Flavor:Google ${METADATA_URL}/attributes/disk-size)
+IMAGE_DESTINATION=$(curl -f -H Metadata-Flavor:Google ${METADATA_URL}/attributes/image-destination)
+ZONE=$(curl -f -H Metadata-Flavor:Google ${METADATA_URL}/zone | cut -d'/' -f4)
+RESERVED_SPACE=50 #50MB reserved space so system can still run
+
+echo "GCECreateDisk: Checking whether image has been created."
```

Comment:
This check is already done by daisy, this script shouldn't need any of this logic